src/button/config.js

```diff
@@ -816,7 +816,9 @@ export const BUTTON_CONFIG : ButtonConfig = {
         allowPrimary: false,
 
         allowPrimaryVertical:   false,
-        allowPrimaryHorizontal: false
+        allowPrimaryHorizontal: false,
+
+        title: `${ FUNDING_BRAND_LABEL.CARD }`
```

Comment:
Is the button component kind of a default component that others override/extend? If so, wondering if setting a default title would cause issues. Just checkin'

Reply:
@gregjopa Is this an issue for v5 as well?

Reply:
@mstuart I noticed this is the same approach that the PayPal button and Credit button are using for aria-label values. Ex: https://github.com/paypal/paypal-checkout-components/blob/v4/src/button/config.js#L212. Unsure if it's causing any issues yet. Still trying to work around getting a successful running local dev environment.

@mnicpt no, this is not an issue in v5. Only applies to v4.

Reply:
@mstuart I got CI passing and added a local dev demo to help with verifying the fix. I think this config change is isolated to just the Debit/Credit Card button container.